packages/react-form-state/src/components/List.tsx

```diff
@@ -45,14 +45,14 @@ export default class List<Fields> extends React.Component<
       const innerFields: FieldDescriptors<Fields> = mapObject(
         fieldValues,
         (value, fieldPath) => {
-          const initialFieldValue = get(initialValue, [index, fieldPath]);
+          const initialFieldValue = initialValue[index][fieldPath];
```

Comment:
This is not a transparent replacement.
`get(initialValue, [index, fieldPath])` this returns `undefined`
whereas 
`initialValue[index][fieldPath];` throws an exception 
if `initialValue[index]` yields undefined